ui/components/app/confirm-data/README.mdx

```diff
@@ -9,7 +9,3 @@ Confirm Data is used on confirmation screen to display transaction data.
   <Story id="components-app-ConfirmData--default-story" />
   <Story id="components-app-ConfirmData--data-component-story" />
 </Canvas>
-
-## Props
-
-<ArgsTable of={ConfirmData} />
```

Comment:
I think we can just remove the README.mdx files altogether. They're only useful if we really want to dig deep on docs for a component otherwise it will generate a basic one. I'll update the storybook docs to define that apologies.

Reply:
Hey I tried to do that, but locally that gives issue @georgewrmarshall